Tests/TuistGraphTests/Models/SwiftPackageManagerDependenciesTests.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+import XCTest
+@testable import TuistGraph
+@testable import TuistSupportTesting
+
+final class SwiftPackageManagerDependenciesTests: TuistUnitTestCase {
+    func test_stringValue_singleDependency() {
```

Comment:
If we rename `stringValue` to `manifestValue`, we should rename the tests here too.